src/components/Kv/KvLightbox.vue

```diff
@@ -130,12 +129,16 @@ export default {
 	methods: {
 		closeLightbox() {
 			if (!this.preventClose) {
-				this.isShown = false;
+				// scroll any body content back to top
+				this.$refs.kvLightboxBody.scrollTop = 0;
```

Comment:
I don't think we want this. If someone has scrolled far down the page and then opens a lightbox, we wouldn't want them to be placed back on the top of the page after they close the lightbox.

Reply:
Nevermind, I see that this is the lightbox body, not the page body, sorry for my mix up!

Reply:
This will only scroll the contents inside the lightbox back up. Their page scroll position won't change.

Reply:
I'll fix that comment